src/sync/installation.ts

```diff
@@ -264,9 +267,18 @@ async function doProcessInstallation(app, data: BackfillMessagePayload, sentry:
 		const { edges, jiraPayload } = await execute();
 		if (jiraPayload) {
 			try {
-				await jiraClient.devinfo.repository.update(jiraPayload, {
-					preventTransitions: true
-				});
+				switch (task) {
```

Comment:
Feature flag?

Reply:
Added feature flag, will add comments at location. ended up being awkward to put in nicely due to the async requirements